src/algorithms/dqns/common.jl

```diff
@@ -35,3 +35,42 @@ function RLBase.update!(
     push!(trajectory[:terminal], is_terminated(env))
     push!(trajectory[:priority], p.learner.default_priority)
 end
+
+#####
+# some common components for Dueling network
+#####
+
+struct DuelingNetwork
```

Comment:
Add a doc string would be better here. Especially, we should document the expected output size of `adv` and `val`.